SingularityBase/src/main/java/com/hubspot/singularity/api/SingularityScaleRequest.java

```diff
@@ -11,16 +11,23 @@
   private final Optional<Boolean> skipHealthchecks;
   private final Optional<Boolean> bounce;
   private final Optional<Boolean> incremental;
+  private final Optional<Boolean> skipEmailNotification;
 
   @JsonCreator
-  public SingularityScaleRequest(@JsonProperty("instances") Optional<Integer> instances, @JsonProperty("durationMillis") Optional<Long> durationMillis,
-      @JsonProperty("skipHealthchecks") Optional<Boolean> skipHealthchecks, @JsonProperty("actionId") Optional<String> actionId, @JsonProperty("message") Optional<String> message,
-      @JsonProperty("bounce") Optional<Boolean> bounce, @JsonProperty("incremental") Optional<Boolean> incremental) {
+  public SingularityScaleRequest(@JsonProperty("instances") Optional<Integer> instances,
+                                 @JsonProperty("durationMillis") Optional<Long> durationMillis,
+                                 @JsonProperty("skipHealthchecks") Optional<Boolean> skipHealthchecks,
+                                 @JsonProperty("actionId") Optional<String> actionId,
+                                 @JsonProperty("message") Optional<String> message,
+                                 @JsonProperty("bounce") Optional<Boolean> bounce,
+                                 @JsonProperty("incremental") Optional<Boolean> incremental,
+                                 @JsonProperty("skipEmailNotification") Optional<Boolean> skipEmailNotification) {
```

Comment:
documenting in-person conversation here as well, we should keep the old constructor around as well so we don't break anyone's builds 👍